test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -8510,7 +8513,7 @@ def cleanup_jobs(self):
                     self.du.run_cmd(host, ['kill', '-9'] + chunk,
                                     runas=ROOT_USER, logerr=False)
             if running_job is True:
```

Comment:
I don't think you need running_job any more now that you have last_running_job.  You can remove the variable.